squashfs-tools/unsquashfs.c

```diff
@@ -2679,6 +2687,8 @@ int main(int argc, char *argv[])
 			ERROR("\t-ll[s]\t\t\tlist filesystem with file "
 				"attributes (like\n");
 			ERROR("\t\t\t\tls -l output), but don't unsquash\n");
+      ERROR("\t-lln[umeric_uid_gid]\tlike -ll[s], but list numeric user and group IDs "
+        "\n");
```

Comment:
The above line should be indented to match surrounding lines, and this line needs further indentation to match nearby continuation lines.